endpoint.cpp

```diff
@@ -394,6 +418,45 @@ int UartEndpoint::open(const char *path, speed_t baudrate)
     return -1;
 }
 
+bool UartEndpoint::_change_baud_cb(void *data)
```

Comment:
this callback is the one that handles the baud_idx and then simply call the `set_speed()` method